subtests/docker_cli/login/login.py

```diff
@@ -161,6 +161,9 @@ def docker_login(self, password=None):
         """
         if password is None:
             password = self.sub_stuff['passwd']
+        # FIXME: --email option is deprecated in 1.12, and will be
+        # removed in 1.13. We still need it for <= 1.10; otherwise
+        # docker prompts from stdin, and we hang.
```

Comment:
Yes, it emits a warning which I only saw by accident. (We really, really need to figure out a way to log all output and capture unexpected warnings). I have no idea what docker 1.13 will do if given the option, or if they really will go through with removing it. I'm just hoping they'll leave it as an undocumented NOP option, hence the FIXME.